portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,122 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Hello, my name is Sydney Beasley</title>
+    <title>Sydney Beasley Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>
+        <intro-title>Hello, my name is Sydney!</intro-title>
```

Comment:
Did you have thoughts about my comments around these extra tags? Feel free to counter my points! Either way, again, not a big deal to change for this project given the timing, but something to think about for future projects.

Reply:
Do you mean your comments to remove the line breaks? If so, it makes sense to remove them, because I'm not trying to break anything but more so add spacing.

Reply:
Hehe, looks like the comment might have gotten lost along the way. The summary is that I think you can remove the custom tags you created and just directly style h1, h2, h3, and button. But again, it's optional for this - enjoy your Friday. I'll copy the original comment below:

---

I'd actually suggest that you don't need the custom tags you've made, particularly intro-title and cursive. Some reasons: 1) in your case, it's simple and direct to just style h1 and h2 in your CSS. h1 already suggests that it's a top-level heading like a title, so having the extra tag to describe it may not be too useful. 2) Regarding the cursive tag, usually it's a good idea not to have anything in your HTML that suggests visual appearance, that way you can choose to change that appearance in the future (let's say by un-cursiving the text) by just updating one CSS rule, and you don't have to go through and change every instance you've added cursive in the HTML.

Reply:
That makes sense! Sorry about so many small changes. Should I go ahead and merge the pull request and make those changes afterwards, or is that bad practice?

Reply:
No worries, in this case, you can go ahead and merge, since I'm happy for the remaining comments to be optional. If you find yourself with extra time in the coming weeks (though I'd be surprised!) you're welcome to try the changes out, but it's fine if you don't make it to them.